js/user-form.js

```diff
@@ -1 +1,24 @@
+const getActivePage = (active) => {
 
+  if (active === false) {
```

Comment:
Так явно проверять лишнее. Если флаг булев то можно делать условие исходя из его значения
```suggestion
  if (active) {
```